subprojects/packagefiles/cjson/include-workaround-meson/cjson/meson.build

```diff
@@ -0,0 +1,9 @@
+# This is a workaround for creating a cjson/ folder with the headers in it in
+# the builddir space.
+#
+# See https://github.com/mesonbuild/meson/issues/2546 for where the technique comes from
+foreach h: headers
+  configure_file(copy: true,
```

Comment:
Should these configure_file() outputs not also be included as sources with the declare_dependency()

Reply:
What would that accomplish? Headers don't need to be added as source files to executable() or library() either.

Reply:
Thought you needed that if you configured a header file that was meant to be public?

Reply:
Why? You just need the include directory, and to guarantee the file is built first. configure_file() is built at setup time, always.

In the specific (non-default) case of headers generated using custom_target or generator, this can be useful to set up an order-only dependency.

Reply:
Sounds good. Thanks